### DISCUSSION TITLE
macOS permission issues

### DISCUSSION THREAD

**Discussion by hollywoodclo** [ptrumpis/snap-camera-server#52] [Q&A] [answered]
I think I’ve got everything working but I’m having an issue on my Mac where it won’t remember any of my permissions for snap camera. Every time I start the app it asks me for permission to connect to my camera and microphone as well as a notification about the application running in the background. Each time I say yes but it always asks me again the next time I start it up. I even went into my settings and that snap camera was listed as a device that’s able to connect to my camera to my microphone, and it is definitely listed. So why does it keep asking me each time again? It’s only this one app. And it doesn’t just ask me for my permission to access the Camera once, it asks me twice upon starting the app, as well as warning me twice that the app is running in the background. And this is all within 20 seconds of asking me the first time. All of these screenshots are things that I had to click on just to start snap camera and it’s kinda of annoying. Anyone know what I did wrong or how to stop this?

<img width="370" alt="Screenshot 2023-08-10 at 10 12 16 PM" src="https://github.com/ptrumpis/snap-camera-server/assets/66051626/3b97cb50-d0b5-4d64-96c1-313d9e9a2cb8">
<img width="370" alt="Screenshot 2023-08-10 at 10 11 23 PM" src="https://github.com/ptrumpis/snap-camera-server/assets/66051626/619c859b-d452-4cd8-87e5-e62259a81cc0">
<img width="370" alt="Screenshot 2023-08-10 at 10 09 14 PM" src="https://github.com/ptrumpis/snap-camera-server/assets/66051626/1e7e8f29-de80-4e36-a992-73cc548f5cf9">
<img width="370" alt="Screenshot 2023-08-10 at 10 11 30 PM" src="https://github.com/ptrumpis/snap-camera-server/assets/66051626/febe88af-3481-436b-a72c-8ec7593652e9">
<img width="370" alt="Screenshot 2023-08-10 at 10 11 17 PM" src="https://github.com/ptrumpis/snap-camera-server/assets/66051626/2136db38-0d34-4fbf-b7e5-b7ea4c8578dc">

**Comment by ptrumpis** [comment, marked as answer]
Did you check your security and privacy settings?
You can try to ask the question at `apple.stackexchange.com` if your privacy settings don't resolve the issue.

There are similar questions [How to stop Terminal from asking for access to some folders?](https://apple.stackexchange.com/questions/388268/how-to-stop-terminal-from-asking-for-access-to-some-folders)
